### DISCUSSION TITLE
TypeScript + Source Maps

### DISCUSSION THREAD

**Discussion by rkarallus** [vercel/next.js#21261] [Help]
We are currently trying to enable source maps for our nextjs typescript project. The goal is to have a stack trace pointing us to the typescript file causing the problem.

What we currently tried:
tsconfig.json => sourceMap: true
next.config.js => productionBrowserSourceMaps: true

In case of an error in the browser, however, we still get the JS stacktrace shown only.

Is there any documentation beyond https://nextjs.org/docs/advanced-features/source-maps, especially with typescript? 

We also found this:
https://github.com/vercel/next.js/discussions/11632
Is the referred plugin https://www.npmjs.com/package/@zeit/next-source-maps still valid and required for next 10?

Thank you very much in advance

**Comment by bramski** [comment]
Ummmm... this doesn't work for me either. What's the deal?

**Comment by azeddine-hmd** [comment]
did anyone find a solution to this ?

**Comment by JulianKingman** [comment]
I'm surprised it's this difficult, this should really be a top priority. There is a `experimental.serverSourceMaps` config, but it doesn't seem to change anything...
